apps/src/music/views/MusicView.jsx

```diff
@@ -182,6 +182,14 @@ class UnconnectedMusicView extends React.Component {
           this.props.clearCallout();
           this.musicBlocklyWorkspace.dispose();
         }
+      })
+      .addListener(LifecycleEvent.LevelLoadCompleted, () => {
+        if (this.props.levelProperties?.appName === 'music') {
+          this.onLevelLoad(
+            this.props.levelProperties?.levelData,
+            this.props.initialSources
```

Comment:
You could even pass the arguments from the lifecycle callback itself since `levelProperties`, `channel`, and `initialSources` are provided as arguments, i.e.
```
.addListener(LifecycleEvent.LevelLoadCompleted, ({appName, levelData}, channel, initialSources) => {
  if (appName === 'music') {
    this.onLevelLoad(levelData, initialSources);
  }
});
```

Reply:
Nice, wil do.